lib/components/artist/artworks/index.js

```diff
@@ -27,81 +23,55 @@ class Artworks extends React.Component {
     const for_sale_count = this.props.artist.counts.for_sale_artworks
     const other_count = this.props.artist.counts.artworks - for_sale_count
     if (for_sale_count === 0) {
-      return this.renderSection('Works',
-                                other_count,
-                                this.props.artist.not_for_sale_artworks,
-                                { availability: 'IS_NOT_FOR_SALE' },
-                                this.resolveQuery,
-                                 null)
+      return this.renderSection({
```

Comment:
I think this is easier to parse, but I've been known to be wrong from time to time.